nzedb/utility/Text.php

```diff
@@ -156,4 +156,58 @@ public static function encodeAsUTF8($data)
 		return $data;
 	}
 
+	/**
+	 * This function turns a roman numeral into an integer
+	 *
+	 * @param string $string
+	 *
+	 * @return int $e
+	 */
+	public static function convertRomanToInt($string) {
+		switch ($string) {
```

Comment:
Is this converted to lower case before calling? If not, wrap the $string in a strtolower().